src/containers.js

```diff
@@ -40,7 +40,13 @@ const CONSTANTS = {
   'fmp4': toUint8([0x73, 0x74, 0x79, 0x70]),
 
   // "ftyp" string literal in hex
```

Comment:
I know it was unchanged,, but I think this should be

```suggestion
  // "ftypqt" string literal in hex
```